tests/unit/lms/services/roster_test.py

```diff
@@ -496,7 +496,7 @@ def test_fetch_assignment_roster_with_lms_api_user_id(
             "Tool does not have access to rlid or rlid does not exist",
         ],
     )
-    def test_fetch_assignment_roster_retries_with_lti_v11_id(
+    def test_fetch_assignment_roster_ignores_known_errors(
```

Comment:
This is the test that test the same thing for assignment rosters, incorrectly named.